nutanix/provider_test.go

```diff
@@ -33,6 +33,13 @@ func TestProvider_impl(t *testing.T) {
 }
 
 func testAccPreCheck(t *testing.T) {
+	if os.Getenv("NUTANIX_USERNAME") == "" ||
+		os.Getenv("NUTANIX_PASSWORD") == "" ||
+		os.Getenv("NUTANIX_INSECURE") == "" ||
```

Comment:
is this a required?, otherwise all seems good to me

Reply:
This is only the function, but we can add this function in each acceptance test case to ensure/required those credentials. This avoids error for missing credencials :)

Reply:
I understand, thank you 👍